store/src/main/java/br/com/estudo/storechallenge/store/request/AddStoreRequest.java

```diff
@@ -0,0 +1,16 @@
+package br.com.estudo.storechallenge.store.request;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+public class AddStoreRequest {
+
+    @NotBlank
```

Comment:
Colocou as anotations do validation, mas não estão sendo usadas. Precisa validar elas, e deve ser feito na entrada. Procura um pouco, o spring tem um jeito bem simples de fazer isso

Reply:
Adicionei a annotation @Valid, porém se precisa de mais algum ajuste ou configuração para aparecer bonitinho quando o request for feito, só me avisar que eu procuro como fazer.